### DISCUSSION TITLE
Search nearest favourites (not possible?)

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#13557] [Q&A] [answered]
How is it possible to show the nearest favourites (around current position OR map-center)?

In "Search" is only history, POI and andresses. In "My Favourites" is only an ALPHABETICLY sorted list of favourites.

![Screenshot_20220116-154815](https://user-images.githubusercontent.com/62500258/149665809-0d438210-4216-4794-880d-9d1221d075ce.png)
![Screenshot_20220116-154834](https://user-images.githubusercontent.com/62500258/149665810-3f6631d2-f0f4-443e-b345-8062b7c91c84.png)

**Comment by pebogufi** [comment, marked as answer]
Favourites:
first, move the position you like th screen center.
tap on magnifier glass right of text "Meine Orte". (second screenshot)
if you want only one group, type in group name.

**Reply by ThomasMeister123** [reply]
Perfect! Thank you for the fast answer!

**Comment by ThomasMeister123** [comment]
Is there the same "search-option" for MARKERS? Cant find it ;(

**Reply by pebogufi** [reply]
Yes, there is a option like ~"more" (translated, might be similar), then "sort by".
Just spend some time to explore the app, there are a lot of performant features.

**Reply by ThomasMeister123** [reply]
Thanx! Yes, sometimes there are TOO MUCH of features ;)
